docs/contact_us.md

```diff
@@ -0,0 +1,19 @@
+# Contact Us
+
+## GitHub
+The preferred point of contact for the Cell Ontology is the GitHub [Issue Tracker](https://github.com/obophenotype/cell-ontology/issues/). It is used commonly for requesting new terms or suggesting changes to existing terms. Feel free to open an issue with any kind of request/question/consideration you might have. 
+
+If you do not have a GitHub account, you can sign up for a free account [here](https://github.com/).
+
+## E-mail and other contacts
+
+The Cell Ontology also has a [Slack Channel](https://anatomy-and-cell-onto.slack.com) and monthly meetings, with an [open agenda](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit). 
+
+If you want to join the channel or talk about any other topics related to the Cell Ontology, you may reach out to one of the active CL editors. 
+The official contact for CL listed at the OBO Foundry is [Alexander Diehl](mailto:addiehl@buffalo.edu), but feel free to contact any other individual. A list of active CL editors can be found [here](http://obophenotype.github.io/uberon/team/).
+
+Note that requests on the [GitHub issue tracker](https://github.com/obophenotype/cell-ontology/issues/) are might be picked up quicker, as individuals are often busy.
```

Comment:
typo 'are' should be removed. My preference would also be to nudge further by using "will probably" heh